modules/installation-configuration-parameters.adoc

```diff
@@ -627,6 +627,12 @@ endif::openshift-origin[]
 |Specify one or more repositories that may also contain the same images.
 |Array of strings
 
+ifdef::aws[]
+|`platform.aws.lbType`
+|Required to set the NLB load balancer type in AWS. Valid values are `Classic` or `NLB`. The default is Classic if no value is provided. The installation program sets the value provided here in the ingress cluster config object. If you do not specify a load balancer type for other ingress controllers, they will use the configuration set in this parameter.
+|Letters `NLB`.
```

Comment:
The values in the 3rd column of the table have to be expressed as values used by the installer program. There's no "Letters" value, it's going to be an ASCII "String" value. I recommend replacing the word "Letters" with "String" and include the word "Classic" also. I recommend the following: "Strings 'Classic' or 'NLB' "